Empty slash:comments tags cause NumberFormatException

When parsing a feed containing an empty `<slash:comments />` tag, Rome throws a `NumberFormatException`.

Example feed: https://rss.golem.de/rss.php?feed=RSS1.0&ms=linux

For posterity's sake, here is an offending `item`:

```xml
    <item rdf:about="https://www.golem.de/news/betriebssysteme-linux-4-19rc1-optimiert-datei-netzwerk-und-cpu-zugriffe-1808-136277.html">
        <dc:format>text/html</dc:format>
        <dc:date>2018-08-30T11:42:00+02:00</dc:date>
        <dc:source>https://www.golem.de</dc:source>
        <dc:creator>Golem.de</dc:creator>
        <title>Betriebssysteme: Linux 4.19rc1 optimiert Datei-, Netzwerk- und CPU-Zugriffe</title>
        <link>https://www.golem.de/news/betriebssysteme-linux-4-19rc1-optimiert-datei-netzwerk-und-cpu-zugriffe-1808-136277.html</link>
        <description>Die Sicherheitslücken in aktuellen Prozessoren lassen die Kernel-Entwickler auch in der kommenden Version 4.19 des Linux-Kernels nicht los. Darüber hinaus gibt es etliche Neuerungen, die Datei- und Netzwerkzugriffe verbessern. (&lt;a href=&quot;https://www.golem.de/specials/linux-kernel/&quot;&gt;Linux-Kernel&lt;/a&gt;, &lt;a href=&quot;https://www.golem.de/specials/virtualisierung/&quot;&gt;Virtualisierung&lt;/a&gt;) &lt;img src=&quot;https://cpx.golem.de/cpx.php?class=17&amp;amp;aid=136277&amp;amp;page=1&amp;amp;ts=1535622120&quot; alt=&quot;&quot; width=&quot;1&quot; height=&quot;1&quot; /&gt;</description>
        <slash:comments />
        <content:encoded><![CDATA[<img src="https://www.golem.de/1808/136277-172404-172403_rc.jpg" width="140" height="140" vspace="3" hspace="8" align="left">Die Sicherheitslücken in aktuellen Prozessoren lassen die Kernel-Entwickler auch in der kommenden Version 4.19 des Linux-Kernels nicht los. Darüber hinaus gibt es etliche Neuerungen, die Datei- und Netzwerkzugriffe verbessern. (<a href="https://www.golem.de/specials/linux-kernel/">Linux-Kernel</a>, <a href="https://www.golem.de/specials/virtualisierung/">Virtualisierung</a>) <img src="https://cpx.golem.de/cpx.php?class=17&amp;aid=136277&amp;page=1&amp;ts=1535622120" alt="" width="1" height="1" />]]></content:encoded>
    </item>
```

This causes the following stack trace in Rome `1.11.0`:

```
com.nononsenseapps.feeder.model.FeedParser$FeedParsingError: com.nononsenseapps.feeder.model.FeedParser$FeedParsingError: java.lang.NumberFormatException: For input string: ""

	at com.nononsenseapps.feeder.model.FeedParser.parseFeedResponse(FeedParser.kt:272)
	at com.nononsenseapps.feeder.model.FeedParserTest.golem1(FeedParserTest.kt:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMainV2.main(AppMainV2.java:131)
Caused by: com.nononsenseapps.feeder.model.FeedParser$FeedParsingError: java.lang.NumberFormatException: For input string: ""
	at com.nononsenseapps.feeder.model.FeedParser.parseFeedInputStream(FeedParser.kt:289)
	at com.nononsenseapps.feeder.model.FeedParser.parseRssAtomBytes$app_debug(FeedParser.kt:279)
	at com.nononsenseapps.feeder.model.FeedParser.parseFeedResponse(FeedParser.kt:255)
	... 30 more
Caused by: java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:592)
	at java.lang.Integer.<init>(Integer.java:867)
	at com.rometools.modules.slash.io.SlashModuleParser.parse(SlashModuleParser.java:61)
	at com.rometools.rome.io.impl.ModuleParsers.parseModules(ModuleParsers.java:52)
	at com.rometools.rome.io.impl.BaseWireFeedParser.parseItemModules(BaseWireFeedParser.java:87)
	at com.rometools.rome.io.impl.RSS090Parser.parseItem(RSS090Parser.java:308)
	at com.rometools.rome.io.impl.RSS10Parser.parseItem(RSS10Parser.java:86)
	at com.rometools.rome.io.impl.RSS090Parser.parseItems(RSS090Parser.java:278)
	at com.rometools.rome.io.impl.RSS090Parser.parseChannel(RSS090Parser.java:184)
	at com.rometools.rome.io.impl.RSS10Parser.parseChannel(RSS10Parser.java:112)
	at com.rometools.rome.io.impl.RSS090Parser.parse(RSS090Parser.java:82)
	at com.rometools.rome.io.WireFeedInput.build(WireFeedInput.java:314)
	at com.rometools.rome.io.WireFeedInput.build(WireFeedInput.java:234)
	at com.rometools.rome.io.SyndFeedInput.build(SyndFeedInput.java:150)
	at com.nononsenseapps.feeder.model.FeedParser.parseFeedInputStream(FeedParser.kt:286)
	... 32 more
```

--- a/rome-modules/src/main/java/com/rometools/modules/slash/io/SlashModuleParser.java
+++ b/rome-modules/src/main/java/com/rometools/modules/slash/io/SlashModuleParser.java
@@ -57,8 +57,8 @@ public class SlashModuleParser implements ModuleParser {
         }
         tag = null;
         tag = element.getChild("comments", SlashModuleParser.NS);
-        if (tag != null) {
-            si.setComments(new Integer(tag.getText()));
+        if (tag != null && !tag.getText().trim().isEmpty()) {
+            si.setComments(new Integer(tag.getText().trim()));
         }
         tag = null;
         tag = element.getChild("department", SlashModuleParser.NS);

